pkg/config/config.go

```diff
@@ -126,7 +126,7 @@ func LoadConfigForRoot(configFileFlagName string, cmdDefaults map[string]interfa
 		case viper.ConfigFileNotFoundError:
 			log.Println("Warning: ", err)
```

Comment:
should this also be changed to an error?

Reply:
Short answer is no. But I think the "real" solution is to refactor this a bit to no longer try and read "aemm-config" as a default config.

 **More detail than you need**
So you'll get this error if you use `viper.SetConfigName()` which is different from when you pass in the config override: `viper.SetConfigFile()`. 

I think `viper.SetConfigName(defaultCfgFileName)` was being used for default values before we started encoding and injecting *aemm-metadata-default-values.json* so right now it's a warning that always triggers unless '-c' param is used. We don't want to make the app unusable because someone doesn't have "aemm-config" in their home dir